Documentation/ceph-cluster-crd.md

```diff
@@ -811,6 +813,7 @@ spec:
       count: 3
       portable: false
       tuneDeviceClass: false
```

Comment:
I'd actually suggest not including the tune device class settings in these documentation examples. They can find them in the doc description, or in the yaml example. We might also consider adding another example in the doc that sets one of these to true, but I'm not sure the scenario is common enough for a full example.

Reply:
Do we want to remove this? As we have an example now.

Reply:
Yes I would still suggest to remove it from these examples.

Reply:
Done.